Sources/Load.php

```diff
@@ -849,6 +850,71 @@ function loadUserSettings()
 	call_integration_hook('integrate_user_info');
 }
 
+function loadMinUserSettings($user_id = 0)
+{
+	global $smcFunc, $cache_enable, $modSettings, $language;
+
+	$user_settings_min = array();
+
+	if (empty($userId) || !is_int($userId))
+		return [];
+
+	if (empty($cache_enable) || ($user_settings_min = cache_get_data('user_settings_min-' . $user_id, 120)) == null)
```

Comment:
Since the logic in this part is not complex,
is the cache checking logic more time consuming as without and
it greate a layer of complexity without need.

Reply:
SMF can be installed in a myriad of server configurations, your unique server env is not a realistic way to determinate if a cache checking is needed or not.

Reply:
This here had nothing do to with my env.
The thing is how many external calls you got and how high the chance is to get a cache hit.

Since every one get one notify is the chance very low that you got a cache hit.
When no cache hit is ther you do a second external call to the database.

So from what i see is that in 90% cases that you slow smf by trying to get data which is not existing,
which result that cached system is slower as a non cached system.

Reply:
Not going to discuss this further unless theres solid data indicating calling cache_get_data() significantly slows down SMF.

Reply:
so you call the cache just for the lulz?